zanata-war/src/main/java/org/zanata/rest/dto/matrix/DetailMatrix.java

```diff
@@ -0,0 +1,20 @@
+package org.zanata.rest.dto.matrix;
+
+import org.zanata.common.ContentState;
+import org.zanata.common.LocaleId;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+/**
+ * @author Patrick Huang
```

Comment:
Some comments explaining this class would be appreciated as its name is a bit generic.